### DISCUSSION TITLE
Formily技术交流群2群【钉钉】

### DISCUSSION THREAD

**Discussion by janryWang** [alibaba/formily#1601] [Ideas]
![IMG_3098](https://user-images.githubusercontent.com/4060976/122532418-10580900-d053-11eb-983d-1c395a6897e4.JPG)

**Comment by haohuawu** [comment]
这个群还能进吗？扫码需要审核，麻烦通过下哈

**Reply by Big-Bear-2019** [reply]
群人满了，进不了了，有什么办法可以加群吗
